frontend/src/tournaments/roundRobin/robinCrosstablePage.tsx

```diff
@@ -0,0 +1,143 @@
+import CohortIcon from '@/scoreboard/CohortIcon';
+import {
+    Box,
+    Card,
+    Container,
+    FormControl,
+    InputLabel,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Typography,
+} from '@mui/material';
+import React, { useEffect, useState } from 'react';
+import {
+    cohorts,
+    fetchTournamentData,
+    fetchTournamentIds,
+    TournamentData,
+} from './roundRobinApi';
+
+const Crosstable: React.FC = () => {
```

Comment:
This doesn't need to be specified as a `React.FC`

Reply:
ok